### ISSUE TITLE
[ASK] How do you get the recommendations from NAML?

### ISSUE THREAD

**Issue by erik-lance** [recommenders-team/recommenders#2036]
### Description

I want to investigate which articles the model recommends. I can't grasp what to do after training because it just shows how to evaluate it unless I am missing something. It outputs a prediction file but it looks like it's supposed to be for the MIND competition? and it predicts everything.

I am wondering if it is possible to predict articles for one user. There doesn't seem to be a clear, usable function, and I don't quite understand the get pred function from the base model.

Basically, can I predict the articles by feeding impression data? or somehow modifying it?

### Other Comments
I want to try using NAML and feed it with my own dataset to study. I am currently figuring out how it might work potentially in production.
